src/utils/logger.js

```diff
@@ -70,29 +70,50 @@ function showHeader(asciiArt, subtitle) {
  * @param {Object} statistics - Statistics about the setup process
  * @param {Object} options - Options for the setup process
  * @param {boolean} options.forceOverwrite - Force overwrite all files regardless of config
+ * @param {boolean} options.dryRun - Whether this was a dry-run
  */
 function showSummary(statistics, options) {
+  // Handle both old and new property names for compatibility
+  const directoriesCreated =
+    statistics.directoriesCreated || statistics.created || 0;
+  const filesCopied = statistics.filesCopied || statistics.copied || 0;
+  const directoriesExisted =
+    statistics.directoriesExisted || statistics.existing || 0;
+  const filesSkipped = statistics.filesSkipped || statistics.skipped || 0;
+  const migrationsApplied = statistics.migrations || 0;
+
   const summaryItems = [
-    ['Directories created', statistics.directoriesCreated],
-    ['Files installed', statistics.filesCopied],
-    [
-      'Existing items preserved',
-      statistics.directoriesExisted + statistics.filesSkipped,
-    ],
+    ['Directories created', directoriesCreated],
+    ['Files installed', filesCopied],
+    ['Existing items preserved', directoriesExisted + filesSkipped],
   ];
 
+  // Add migrations if any were applied
+  if (migrationsApplied > 0) {
+    summaryItems.push(['Migrations applied', migrationsApplied]);
+  }
+
   showLine();
-  console.log(style.bold(style.success(' ✨ AWOS Setup Complete!')));
-  console.log('');
-  console.group(style.bold('  Summary:'));
+
+  if (options?.dryRun) {
+    console.log(style.bold(style.warn(' 🔍 Dry-Run Complete')));
+    console.log('');
+    console.log(style.dim('  This was a preview. No files were modified.'));
+    console.log('');
+    console.group(style.bold('  What would happen:'));
+  } else {
+    console.log(style.bold(style.success(' ✨ AWOS Setup Complete!')));
+    console.log('');
+    console.group(style.bold('  Summary:'));
+  }
 
   summaryItems.forEach(([label, value]) => {
-    console.log(
-      `${style.success('•')} ${label}: ${style.bold(value.toString())}`
-    );
+    const prefix = options?.dryRun ? style.dim('○') : style.success('•');
+    const labelText = options?.dryRun ? `Would ${label.toLowerCase()}` : label;
```

Comment:
```suggestion
    const labelText = label;
```

`What would happen:` as a header already ok.   

```
  ○ Would directories created: 0
  ○ Would files installed: 19
  ○ Would existing items preserved: 12
  ○ Would migrations applied: 1
```
Sounds incorrect

Reply:
Fixed phrasing: 
```
  - ○ Would create directories: 5
  - ○ Would install files: 26
  - ○ Would preserve existing items: 0
  - ○ Would apply migrations: 1
```